File: client/bcephal-client-model/Grids/Filters/FindReplaceFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Grids.Filters
{
    public class FindReplaceFilter
    {

		public BrowserDataFilter Filter { get; set; }

		public GrilleColumn Column { get; set; }

		public bool IgnoreCase { get; set; }

		public bool ReplaceOnlyValuesCorrespondingToCondition { get; set; }

		public String Operation { get; set; }

		public String Criteria { get; set; }
		public decimal? MeasureCriteria { get; set; }

		public String ReplaceValue { get; set; }
		public decimal? MeasureReplaceValue { get; set; }

	}
}

File: server/bcephal-grid/src/main/java/com/moriset/bcephal/grid/domain/FindReplaceFilter.java
/**
 * 
 */
package com.moriset.bcephal.grid.domain;

import java.math.BigDecimal;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @author Joseph Wambo
 *
 */
@Data
@EqualsAndHashCode(callSuper = false)
public class FindReplaceFilter {

	private GrilleDataFilter filter;

	private GrilleColumn column;
    
	private boolean ignoreCase;
	
	private boolean replaceOnlyValuesCorrespondingToCondition;

	private String operation;

	private String criteria;
	private BigDecimal measureCriteria;

	private String replaceValue;
	private BigDecimal measureReplaceValue;
	
}
